README.md

```diff
@@ -1,3 +1,14 @@
+
+## Note
+
+Please consider using native promises instead of Q. Native promises are faster, have better tooling support and are the future.
```

Comment:
```suggestion
Please consider using [JavaScript promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) instead of Q. Native promises are faster, have better tooling support and are the future.
```